dashboard/app/models/lesson.rb

```diff
@@ -291,6 +291,17 @@ def summarize(include_bonus_levels = false, for_edit: false)
     lesson_summary.freeze
   end
 
+  def summarize_for_calendar
+    {
+      id: id,
+      title: localized_title,
+      duration: lesson_activities.map(&:summarize).sum {|activity| activity[:duration] || 0},
+      assessment: !!assessment,
+      unplugged: script.is_migrated ? unplugged : display_as_unplugged, # TODO: Update to use unplugged property
```

Comment:
Hey Jillian - I got the update for unplugged lessons done which will fix this line. Its probably going to get merged today (https://github.com/code-dot-org/code-dot-org/pull/39039). So I wanted to give you the heads up and let you know that this line should become 

```
unplugged: unplugged
```